packages/app/config/stubs/lru-cache.js

```diff
@@ -1,482 +0,0 @@
-'use strict';
```

Comment:
Why did you delete this? 🤔

Reply:
I am trying to add this file again but then it is failing on pre-commit hook!